### PR TITLE
Add hotspot_custom for hotspot component 

### LINKED ISSUE #2878: Add hotspot_custom for hotspot component
It would be nice to add hotspot specific custom testcase, which would enable us to run a specific hotspot test or hostspot tests under specific path easily in jenkins grinder.

Adding hotspot_custom is really similar to add langtools_custom testcase, https://github.com/adoptium/aqa-tests/issues/2692#issuecomment-870844104 .

### PR DESCRIPTION
**Pull request by zahraaliaghazadeh**
Fixes #2878

### CODE REVIEW

File: openjdk/playlist.xml [hunk 1]
```diff
@@ -36,6 +36,30 @@
...
+			<variation>Mode650</variation>
+			<variation>Mode1000</variation>
+		</variations>
+		<command>$(JAVA_COMMAND) -Xmx512m -jar $(Q)$(TEST_RESROOT)$(D)jtreg$(D)lib$(D)jtreg.jar$(Q) \
+	$(JTREG_BASIC_OPTIONS) $(CUSTOM_NATIVE_OPTIONS) -vmoptions:$(Q)-Xmx512m $(JVM_OPTIONS) $(VMOPTION_HEADLESS)$(Q) \
+	$(TIMEOUT_HANDLER) \
+	-w $(Q)$(REPORTDIR)$(D)work$(Q) \
+	-r $(Q)$(TEST_RESROOT)$(D)report$(Q) \
+	-jdk:$(Q)$(TEST_JDK_HOME)$(Q) \
+	$(Q)$(JTREG_HOTSPOT_TEST_DIR)$(D)$(JDK_CUSTOM_TARGET)$(Q); \
```
Comment (sophia-guo) [inline openjdk/playlist.xml:53]: JDK_CUSTOM_TARGET need to be updated to HOSTSPOT_CUSTOM_TARGET.

**Comment by sophia-guo** [comment]
Add the default testcase please see https://github.com/adoptium/aqa-tests/issues/2878#issuecomment-932631300

**Comment by smlambert** [comment]
Test via: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/1543/

**Comment by sophia-guo** [comment]
Test via: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/1544/console

**Review by sophia-guo — APPROVED** [review]
LGTM. Thanks @zahraaliaghazadeh

**Pull request merged**